packages/extester/src/cli.ts

```diff
@@ -64,8 +64,9 @@ program.command('setup-tests')
     .option('-t, --type <type>', 'Type of VS Code release (stable/insider)')
     .option('-y, --yarn', 'Use yarn to build the extension via vsce instead of npm', false)
     .option('-i, --install_dependencies', 'Automatically install extensions your extension depends on', false)
+    .option('-d, --extension_development_path <extension development directory>', 'VSCode will use the unpackaged extension source from this directory')
```

Comment:
I would rather go without this new `extension_development_path` argument.. the current process is to run tests automatically against packed VSIX from the current working directory

for coverage I would like to just provide new `--coverage` option which will in the backend also used the `process.cwd()` to get project files

could you please remove `extension_development_path` where it is not needed and use for coverage directly the cwd?

Reply:
I see. I will make the changes accordingly. Note that still the `--extensionDevelopmentPath` option needs to be specified in `VSBrowser` class:
```
        } else if(this.extensionDevelopmentPath) {
           args.push(`--extensionDevelopmentPath=${this.extensionDevelopmentPath}`);
        }
```
and code will be loaded from there instead of the installed .vsix for getting coverage, but if we can assume extension development path is always current directory, we do not need to add the argument for code coverage support.

Reply:
One thing I want to add before releasing this feature is the support for config file (`.nycrc`, etc.). For instance, some users may want to save reports in a directory that is different from the default (`./coverage`). `c8` has [a smart way](https://github.com/bcoe/c8/blob/main/lib/parse-args.js#L21) to implement such config support and I would like to add similar codes for that.

Reply:
ok, support that config file sounds good

Reply:
It is implemented in `coverage.ts`.  Four JSON config file names '.c8rc', '.c8rc.json', '.nycrc', '.nycrc.json' are supported (=same as c8).